dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -63,29 +60,10 @@ public class TeradataLogsConnector extends AbstractTeradataConnector
 
   private static final Logger LOG = LoggerFactory.getLogger(TeradataLogsConnector.class);
 
-  @VisibleForTesting /* pp */ static final String DEF_LOG_TABLE = "dbc.DBQLogTbl";
-  /* pp */ static final String ASSESSMENT_DEF_LOG_TABLE = "dbc.QryLogV";
```

Comment:
This constant is used only in this class, so it should not be moved to `AbstractTeradataConnector`.